languages/js/src/errors.ts

```diff
@@ -129,3 +129,16 @@ export class DataFilteringConfigurationError extends PolarError {
     );
   }
 }
+
+export class UnexpectedPolarTypeError extends PolarError {
+  constructor(message: string) {
+    super(`${message}`);
+  }
+}
+
+export const UNEXPECTED_EXPRESSION_MESSAGE = `Received Expression from Polar VM. The Expression type is only supported when
+  using data filtering features. Did you perform an
+  operation over an unbound variable in your policy?
+
+  To silence this error and receive an Expression result, pass
+  accept_expression as true to Oso.Query.`;
```

Comment:
Since we only construct this error in a single place, I'd prefer to either:
- Raise a `PolarError` w/ `UNEXPECTED_EXPRESSION_MESSAGE` as the `message`, removing the new `UnexpectedPolarTypeError` class
- Rename `UnexpectedPolarTypeError` -> something like `UnexpectedExpressionError` and directly call `super(UNEXPECTED_EXPRESSION_MESSAGE);` in the `constructor`

Reply:
I decided to go with option 2 since I think it differentiates the error nicely from a generic PolarError. Lmk if you want to change it!